### DISCUSSION TITLE
Where can I know the available triplets of libraries now?

### DISCUSSION THREAD

**Discussion by trend-bai-chuan-huang** [microsoft/vcpkg#38709] [Q&A] [answered]
I want to know the available triplets of libraries, like arm64-windows, x64-windows, x64-windows-static for gtest library.

The info could be found on the vcpkg sites in the last week.

But now, I cannot get the info of available triplets on the [vcpkg site](https://vcpkg.io/en/packages)

 Where can I  get this info or which command  should I type?
 
 Thank you

**Comment by dg0yt** [comment, marked as answer]
You can always check the `<triplet>.cmake` filenames in this location:
https://github.com/microsoft/vcpkg/tree/master/triplets

**Reply by dg0yt** [reply]
... including the `community` subdir.
